### ISSUE TITLE
.flac /.cue Album support 

### ISSUE THREAD

**Issue by BarbaricBeefcake** [MusicPlayerDaemon/MPD#313]
Hello!
I am using the Volumio distribution, which seems to run MPD as its music player. 

So, most of my music are stored as one flac file with a .cue file and not as single tracks.

If i want to play a single song that I added to the player queue, MPD won't jump the next song after the previous song was finished.
MPD keeps on playing the whole rest of the flac file until the end and plays then what is netxt in queue.

So, basically I'm only able to play the whole "CD" and not single tracks from different albums in a queue. 

Could this be fixed somehow?

Thanks, 
JayZee

**Comment by MaxKellermann** [comment]
No, Volumio doesn't use MPD.
MPD doesn't have the problem you described.
